test/SmokeTests/OOProcSmokeTests/DotNetIsolated/Counter.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Runtime.Serialization;
+using System.Text;
+using Azure.Core;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Extensions.DurableTask;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.DurableTask;
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.Entities;
+using Microsoft.Extensions.Logging;
+
+namespace DotNetIsolated;
+
+
+/// <summary>
+/// A simple counter, demonstrating entity use.
+/// </summary>
+[DataContract]
```

Comment:
Data attributes are not necessary. They might even be misleading; STJ does not support them by default. You have to implement a completely custom `STJ.DataContractResolver`. Would be better to remove and let customers figure out serialization through existing STJ examples/docs.

Reply:
The goal was to clarify to the reader that this class is going to be serialized when saving the entity state.
Its generally something I like to make very clear, even if attributes are not technically required.

What is the "STJ way" to accomplish this? Also, what is STJ?

Reply:
(For now I have just removed this)

Reply:
I think STJ is `System.Text.JSON`